modules/installation-ibm-cloud-iam-policies-api-key.adoc

```diff
@@ -64,14 +73,87 @@ You must assign the required access policies to your IBM Cloud account.
 3. Only one service is required. The service that is required depends on the type of cluster that you are installing. If you are installing a public cluster, `Internet Services` is required. If you are installing a private cluster, `DNS Services` is required.
 --
 //TODO: IBM confirmed current values in the table above. They hope to provide more guidance on possibly scoping down the permissions (related to resource group actions).
+endif::ibm-vpc[]
+
+ifdef::ibm-power-vs[]
+[id="pre-requisite-permissions-ibm-cloud_{context}"]
+== Pre-requisite permissions
+
+.Pre-requisite permissions
+[cols="1,2",options="header"]
+|===
+|Role |Access
+
+|Viewer, Operator, Editor, Administrator, Reader, Writer, Manager
+|Internet Services service in <resource_group> resource group
+
+|Viewer, Operator, Editor, Administrator, User API key creator, Service ID creator
+|IAM Identity Service service
+
+|Viewer, Operator, Administrator, Editor, Reader, Writer, Manager, Console Administrator
+|VPC Infrastructure Services service in <resource_group> resource group
+
+|Viewer
+|<resource_group> resource group: resourceType string equals resource-group, resource string equals <resource_group>
+|===
+
+[id="cluster-creation-permissions-ibm-cloud_{context}"]
+== Cluster-creation permissions
+
+.Cluster-creation permissions
+[cols="1,2",options="header"]
+|===
+|Role |Access
+
+|Viewer
+|<resource_group> (Resource Group Created for Your Team)
+
+|Viewer, Operator, Editor, Reader, Writer, Manager
+|All service in Default resource group
+
+|Viewer, Reader
+|Internet Services service
+
+|Viewer, Operator, Reader, Writer, Manager, Content Reader, Object Reader, Object Writer, Editor
+|Cloud Object Storage service
+
+|Viewer
+|Default resource group: resourceType string equals resource-group, resource string equals Default
```

Comment:
The above content was provided in this comment : https://github.com/openshift/openshift-docs/pull/47208#issuecomment-1176792430
@clnperez request you to review above content and please let me know if you agree to the above suggestions. 
I agree on to add backticks. Thanks.

Reply:
for this one, instead of 
_Default resource group: resourceType string equals resource-group, resource string equals Default_

`Default` Resource group: : Access to view the `Default` `Resource group` itself (as opposed to resources it controls). The resource type should equal Resource group, with a value of `Default`. If your account administrator changed your account's default resource group to something other than `Default`, use that value instead.

Reply:
Updated that stated above. Thanks @clnperez.
cc : @stevsmit